### DISCUSSION TITLE
How to fix plot_pca_correlation_graph Import issue on Colab?

### DISCUSSION THREAD

**Discussion by kbnnf** [rasbt/mlxtend#940] [Q&A]
#### Describe the bug

Hi, i have an issue import plot_pca_correlation_graph on Colab, i tried !pip install and !git clone mlxtend, also tried to install the dev version but for some reason it's not working either (i already managed to install the dev version in Colab for another projet).

I tried importing plot_decision_regions and that one works.

#### Steps/Code to Reproduce

```python
!pip install mlxtend
from mlxtend.plotting import plot_pca_correlation_graph
```

#### Actual Results
ImportError: cannot import name 'plot_pca_correlation_graph' from 'mlxtend.plotting' (/usr/local/lib/python3.7/dist-packages/mlxtend/plotting/__init__.py)

#### Versions

MLxtend 0.14.0
Linux-5.4.188+-x86_64-with-Ubuntu-18.04-bionic
Python 3.7.13 (default, Apr 24 2022, 01:04:09) 
[GCC 7.5.0]
Scikit-learn 1.0.2
NumPy 1.21.6
SciPy 1.4.1

**Comment by rasbt** [comment]
Hi there. Just checking the [release notes](http://rasbt.github.io/mlxtend/CHANGELOG/), the function was added in version 0.17, and it looks like your version is 0.14. Upgrading to the recent mlxtend version will probably fix the issue!

Btw I think you need 

    !pip install mlxtend --upgrade

**Comment by rasbt** [comment]
Btw I think you need 

    !pip install mlxtend --upgrade
